### ISSUE TITLE
Failure of normal back-and-forth conversation

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7273]
**Preferred term label:**
Failure of normal back-and-forth conversation

**Synonyms**

**Definition (free text, please give PubMed ID)**
A deficit in normal social reciprocity. (DSM-V, pg 50)

**Parent term (use hpo.jax.org/app)**
Impaired social reciprocity (HP:0012760)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Autism Spectrum Disorder

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
The definition is only partially correct because "A deficit in normal social reciprocity" can include things other than conversation? One rule for the HPO definitions is that in general they should be comprehensible without context. The above definition might not be comprehensible for somebody who is not reading the DSM-V. Perhaps adding some examples would also help.

**Comment by pnrobinson** [comment]
Label: Poor conversational reciprocity
Synonym: Failure of normal back-and-forth conversation
Def: Reduced ability to conduct a reciprocal (back and forth) conversation characterized by deficits in the ability to make comments about discussion topics of others, failure to ask questions, and not building a conversation on comments of others. Affected persons may not know what to say next in a conversation, may not notice if others are bored with a given topic, may conduct a one-sided conversation (talking but not listening), or may veer off topic more than is socially acceptable.

**Comment by pnrobinson** [comment]
added as HP:4000092
